generator/ServiceClientGeneratorLib/Generators/Endpoints/EndpointResolver.partial.cs

```diff
@@ -24,7 +24,7 @@ private string GetValueSource(Parameter parameter)
                 case "AWS::S3Control::UseArnRegion": return "config.UseArnRegion";
                 case "AWS::S3::DisableMultiRegionAccessPoints": return "config.DisableMultiregionAccessPoints";
                 case "AWS::S3::UseGlobalEndpoint": return "config.USEast1RegionalEndpointValue == S3UsEast1RegionalEndpointValue.Legacy";
-                case "AWS::STS::UseGlobalEndpoint": return "config.StsRegionalEndpoints == StsRegionalEndpointsValue.Legacy";
+                case "AWS::STS::UseGlobalEndpoint": return "false";
```

Comment:
I had to keep this built-in otherwise the generator would fail. It's only used in the STS endpoint rules: https://github.com/aws/aws-sdk-net/blob/cb7de169cc3666096b9f38b2cb13d30404d258cc/generator/ServiceModels/sts/sts-2011-06-15.endpoint-rule-set.json#L30-L36

I'll double check if there's any other impact, there are endpoint rules which set `"AWS::STS::UseGlobalEndpoint": true` but all of our generated tests are still passing.